website/docs/introduction/getting-started/first.md

```diff
@@ -115,6 +115,8 @@ $ kubectl wait --for=condition=Ready pods --all -n catalog --timeout=180s
 
 Now that the Pods are running we can [check their logs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs), for example the catalog API:
 
+Note:  You can ["follow" the kubectl logs output](https://kubernetes.io/docs/reference/kubectl/cheatsheet/) by using the '-f' option with the command.  (Use CTRL-C to stop following the output)
```

Comment:
Would it make sense to wrap this in a "tip" admonition and remote the "Note:"?

https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Good call - thx @niallthomson - I'll clean it up now